### ISSUE TITLE
Upload pre-trained models weights 

### ISSUE THREAD

**Issue by fahad-shamshad** [shaohua0116/DCGAN-Tensorflow#1]
Kindly upload pre-trained models weights of DCGAN-Tensorflow for MNIST fashion, DIGITS, SVHN and CIFAR-10. Thanks.

**Comment by shaohua0116** [comment]
It would be really easy to reproduce the results by tuning the learning rate and the update rate a little bit. I actually deleted all the checkpoints after testing the model. Please note that I am actually not responsible for providing the checkpoints. Just for this time, I trained all the models on the four datasets again and pushed the checkpoints and the corresponding results under the directory `./checkpoint/`.
